sources/packages/api/src/services/sin-validation/sin-validation.service.ts

```diff
@@ -55,13 +57,13 @@ export class SINValidationService extends RecordDataModelService<SINValidation>
         "sinValidation.temporarySIN",
         "sinValidation.sinExpiryDate",
       ])
-      .where("sinValidation.user.id = :userId", { userId })
+      .where("sinValidation.student.id = :studentId", { studentId })
```

Comment:
As possible, we can take these opportunities to convert string-based queries to the new type-safe Typeorm model.